xc7/archs/artix7/devices/CMakeLists.txt

```diff
@@ -1,5 +1,12 @@
 add_xc7_device_define(
   ARCH artix7
+  USE_ROI
   DEVICES xc7a50t-basys3 xc7a50t-arty-uart xc7a50t-arty-swbut
   PARTS xc7a35tcpg236-1 xc7a35tcsg324-1 xc7a35tcsg324-1
 )
+add_xc7_device_define(
+  ARCH artix7
+  DEVICES xc7a50t
+  PARTS xc7a35tcpg236-1
+  GRAPH_LIMIT 61,105,114,155
```

Comment:
I don't like this format very much -- but I assume it is only temporary? If so I don't care that much...

Reply:
Ya, it's just temporary.  This will go away after we support the full 50T routing graph.